eZ/Publish/Core/Base/Container/Compiler/Search/SignalSlotPass.php

```diff
@@ -20,31 +19,47 @@
  */
 class SignalSlotPass implements CompilerPassInterface
 {
+    /**
+     * Name of a search engine for which Compiler Pass was instantiated.
+     *
+     * @var string|null
+     */
+    private $searchEngineAlias;
+
+    public function __construct($searchEngineAlias = null)
```

Comment:
@andrerom: I wanted this to be a soft change, to avoid breaking Solr: https://github.com/ezsystems/ezplatform-solr-search-engine/blob/master/bundle/EzSystemsEzPlatformSolrSearchEngineBundle.php#L36